database/src/main/resources/changesets/metadata_journal.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
+
+    <changeSet author="mcovarr" id="metadata_journal">
+        <createTable tableName="METADATA_JOURNAL">
+            <column autoIncrement="true" name="METADATA_JOURNAL_ID" type="INT">
+                <constraints primaryKey="true" primaryKeyName="PK_METADATA_JOURNAL"/>
+            </column>
+            <column name="WORKFLOW_EXECUTION_UUID" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
```

Comment:
Yup that's pretty much the idea, we explicitly don't want referential integrity checking.  Yesterday this branch was using WORKFLOW_EXECUTION_ID as a regular FK, but per discussion with @geoffjentry and @cjllanwarne this was refactored to the current model.  The calling APIs will at least enforce a WorkflowId is passed, but I'm not sure the construction of those is restricted in any meaningful way.